### PR TITLE
Added the Redis 7.4 GA release

### PR DESCRIPTION
**Pull request by dmaier-redislabs**
Removed 7.4 RC2 and added 7.4 GA

### CODE REVIEW

**Comment by dmaier-redislabs** [comment]
Hi @tianon,

we just released the GA version of 7.4. Could someone on your side please merge this PR in?

Thanks in advance, 
David

File: library/redis [hunk 1]
```diff
@@ -1,18 +1,18 @@
-# this file is generated via https://github.com/redis/docker-library-redis/blob/ccb36ac7abb297eed89a517a02f9fbdb74d2a015/generate-stackbrew-library-mac.sh
+# This file was generated via https://github.com/redis/docker-library-redis/blob/e92049967d6baa9b4b3d6fd950553e6458e61f13/generate-stackbrew-library-mac.sh
 
 Maintainers: David Maier <david.maier@redis.com> (@dmaier-redislabs),
              Yossi Gottlieb <yossi@redis.com> (@yossigo)
 GitRepo: https://github.com/redis/docker-library-redis.git
 
-Tags: 7.4-rc2, 7.4-rc, 7.4-rc2-bookworm, 7.4-rc-bookworm
+Tags: 7.4.0, 7.4, 7.4.0-bookworm, 7.4-bookworm
```
Comment (tianon) [inline library/redis:7]: You probably also want to update the generic `7` and `latest` aliases to point here:

https://github.com/redis/docker-library-redis/blob/e92049967d6baa9b4b3d6fd950553e6458e61f13/generate-stackbrew-library-mac.sh#L11

(that'll move `alpine`, etc too)

Comment (dmaier-redislabs) [reply library/redis:7]: You are right. Let me fix that.

Comment (dmaier-redislabs) [reply library/redis]: OK, this should be fixed now.

**Review by yosifkit — APPROVED** [review]

**Pull request merged**